adr/ADR-8.md

```diff
@@ -164,6 +164,9 @@ type KV interface {
 	// Purge removes all data for a key including history, leaving 1 historical entry being the purge
 	Purge(key string) error
 
+	// Completely remove all keys that are in the deleted or purged state, leaving no entries behind for those keys.
+	PurgeDeletes() error
```

Comment:
We decided not to add this to the required API. So doesn’t really need to be here or should be marked as optional

Reply:
Ok I'll make it optional so we can capture the comment as to what it does